modules/user-guide/partials/benefits-of-PR-review-in-Che.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: PROCEDURE
+
+[id="title_{context}"]
+= Benefits of reviewing pull and merge requests in {prod}
+
+[role="_abstract"]
+{prod} workspace contains all tools you need to review pull and merge requests from start to finish. By clicking a {prod-short} link, you get access to {prod}-supported web IDE with a ready-to-use workspace where you can run a linter, unit tests, the build and more.
+
+.Prerequisites
+* You have access to the repository hosted by your Git provider.
+* You use a {prod}-supported browser: Google Chrome or Mozilla Firefox.
+* You have access to a {prod-short} instance.
+
+.Procedure
+. Open the feature branch to review in {prod-short}. A clone of the branch opens in a workspace with tools for debugging and testing.
+. Check the pull or merge request changes.
+. Run your desired debugging and testing tools:
+** Run a linter.
+** Run unit tests.
+** Run the build.
+** Run the application to check for problems.
+. Navigate to UI of your Git provider to leave comment and pull or merge your assigned request.
+
+.Verification
+
+. (optional) Open a second workspace using the main branch of the repository to reproduce a problem.
```

Comment:
```suggestion
. Optional: Open a second workspace using the main branch of the repository to reproduce a problem.
```
This is a tip I learned from the OpenShift docs team. Any optional procedure steps they mark as `. Optional: [Start sentence with a capital letter]`